source/material_model/equation_of_state/multicomponent_compressible.cc

```diff
@@ -147,6 +223,24 @@ namespace aspect
 
         options.property_name = "Isochoric specific heats";
         isochoric_specific_heats = Utilities::MapParsing::parse_map_to_double_array (prm.get(options.property_name), options);
+
+        enable_phase_transitions = prm.get_bool ("Enable phase transitions");
+
+        // Check that phase transitions have explicitly been enabled if the number of phases
+        // for any compositional field is greater than 0
+        if (!enable_phase_transitions)
+          {
+            for (unsigned int n=0; n < (*expected_n_phases_per_composition).size(); ++n)
+              {
+                AssertThrow((*expected_n_phases_per_composition)[n] == 1, ExcMessage(
+                              "The number of expected phases per composition for the compositional "
```

Comment:
Can you make that more readable?
```suggestion
                AssertThrow((*expected_n_phases_per_composition)[n] == 1, 
                              ExcMessage("The number of expected phases per composition for the compositional "
```